src/modules/top-level.nix

```diff
@@ -25,6 +25,18 @@ in
       default = { };
     };
 
+    beforeUp = lib.mkOption {
```

Comment:
this would be better named `process.before` in process.nix

Reply:
done :)